frontend/vite.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import viteTsconfigPaths from 'vite-tsconfig-paths';
+import svgrPlugin from 'vite-plugin-svgr';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react(), viteTsconfigPaths(), svgrPlugin()],
+  server: {
+    port: 8080,
```

Comment:
either change this to `port: 3000` or the frontend README TO `ngrok http 8080`

Reply:
ok! i realise my original env port was wrong lol